include/fi_util.h

```diff
@@ -330,7 +330,7 @@ struct util_event {
 	uint8_t			data[0];
 };
 
-int fi_eq_create(struct fid_fabric *fabric, struct fi_eq_attr *attr,
+int ofi_eq_create(struct fid_fabric *fabric, struct fi_eq_attr *attr,
 		 struct fid_eq **eq_fid, void *context);
```

Comment:
Please search the source tree for the old names (util_trywait, fi_wait_fd_open, fi_eq_create) to make sure that the renames won't break the build for some other provider.